parametric/test_headers_b3.py

```diff
@@ -57,8 +55,8 @@ def test_headers_b3_extract_invalid(test_agent, test_library):
 
 @enable_b3()
 @pytest.mark.skip_library("dotnet", "Latest release does not implement new configuration")
-@pytest.mark.skip_library("golang", "not impemented")
-@pytest.mark.skip_library("nodejs", "not impemented")
+@pytest.mark.skip_library("nodejs", "not implemented")
```

Comment:
```suggestion
```

Reply:
I was just fixing a typo but okay 😅

Reply:
oh wait, I think this was my bad lol. For some reason it looked like we added another `nodejs` entry. All good for the typo fix lol